From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Liav A <liavalb@gmail.com>
Date: Fri, 27 May 2022 22:00:38 +0300
Subject: [PATCH] Fix printf bad specifier formatting

Fix sprintf specifier being written in a bad format leading to iASL to
crash.
---
 source/common/dmtable.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/common/dmtable.c b/source/common/dmtable.c
index 440c5fb..d722e3d 100644
--- a/source/common/dmtable.c
+++ b/source/common/dmtable.c
@@ -977,7 +977,7 @@ AcpiDmLineHeader (
     {
         if (ByteLength)
         {
-            AcpiOsPrintf ("[%3.3Xh %4.4d% 4d] %28s : ",
+            AcpiOsPrintf ("[%3.3Xh %4.4d %4d] %28s : ",
                 Offset, Offset, ByteLength, Name);
         }
         else
@@ -1011,7 +1011,7 @@ AcpiDmLineHeader2 (
         }
         else
         {
-            AcpiOsPrintf ("%36s % 3d : ",
+            AcpiOsPrintf ("%36s %3d : ",
                 Name, Value);
         }
     }
